testrepository/tests/test_ui.py

```diff
@@ -251,3 +261,23 @@ def test_make_result_previous_run(self):
         result.status()
         result.stopTestRun()
         summary.wasSuccessful()
+
+    def test_output_tests_meta_unknown_style(self):
+        ui = self.get_test_ui()
+        with ExpectedException(Exception, 'unknown style'):
+            ui.output_tests_meta([], style='strange')
+
+class TestOutputTestsMeta(TestUIContractBase):
+
+    scenarios = multiply_scenarios(ui_implementations, [
+        ('json', dict(style='json')),
+        ('list', dict(style='list')),
+        ])
+
+    def test_output_tests_meta(self):
```

Comment:
I personally prefer longer, more descriptive test ids: 'test_output_tests_meta' doesn't really hint at what the intent of this test is. How about:

test_ui_output_tests_meta_accepts_style_argument

...or similar?